modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/persistence/RocksDbKeyValueStorage.java

```diff
@@ -1465,23 +1467,23 @@ private void replayUpdates(long lowerRevision, long upperRevision) {
 
                 if (revision != lastSeenRevision) {
                     if (!updatedEntries.isEmpty()) {
-                        var updatedEntriesCopy = List.copyOf(updatedEntries);
+                        var updatedEntriesCopy = (List<Entry>) List.copyOf(updatedEntries);
 
-                        assert ts != null;
+                        assert ts != null : revision;
 
                         watchProcessor.notifyWatches(updatedEntriesCopy, ts);
 
                         updatedEntries.clear();
 
-                        ts = timestampByRevision(revision);
```

Comment:
Do we still need the method itself?

Reply:
At the moment, no, but it may be needed later, if not, we will get rid of it later.